src/sentry/tasks/relay.py

```diff
@@ -99,6 +88,13 @@ def update_config_cache(
 
         projectconfig_cache.delete_many(cache_keys_to_delete)
 
+    # After this new tasks for this debounce key can be scheduled again.
+    projectconfig_debounce_cache.mark_task_done(
+        organization_id=organization_id,
+        project_id=project_id,
+        public_key=public_key,
+    )
```

Comment:
There's an early return on line 67 for which we should `mark_task_done` as well. Maybe we should wrap the function body in a `try ... finally` block to be sure the task is marked as done?

Or even turn `mark_task_done` into a context manager?